Lesson_Materials/Lecture_08_Using_USM/index.html

```diff
@@ -168,7 +168,7 @@
 						<code class="code-100pc"><pre>
 int square_number(int x){
 	
-  auto myQueue = sycl::queue{};
+  auto myQueue = queue{};
```

Comment:
Is it clearer to remove the namespace?

Reply:
It's more consistent since nothing else in the snippet has the namespace. Could always add a namespace to the others